test/namenode/NameNodeTest.cc

```diff
@@ -0,0 +1,110 @@
+#include "zkwrapper.h"
+#include "zk_nn_client.h"
+
+#include <gtest/gtest.h>
+
+#include <cstring>
+#include <easylogging++.h>
+
+INITIALIZE_EASYLOGGINGPP
+
+namespace {
+
+	class NamenodeTest : public ::testing::Test {
+
+	protected:
+		virtual void SetUp() {
+			system("sudo ~/zookeeper/bin/zkServer.sh stop");
```

Comment:
@prb2 can you try structuring it like this? https://github.com/Rice-Comp413-2016/RDFS/blob/addBlockMultiOp/test/zkwrapper/ZKWrapperTest.cc